common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisControlManager.java

```diff
@@ -290,6 +359,46 @@ public void onFlightEnd() {
         TardisEvents.LAND.invoker().onLand(operator, lastKnown.level, lastKnown.position);
     }
 
+    // Triggers the crash event.
+    public void crash() {
```

Comment:
Do we want to add an event trigger for crash start and crash end? It may be useful for other mods to add some behaviour.

Reply:
Any suggestions for what the event might contain?

Reply:
I agree with a event for crashing. Should give player access to the Capability and crash position